canonical/ubuntu-pro-client
# ci: check that new changelog entries get all bug references

## Why is this needed?

This will hopefully help me remember to include all of the bug references in new release changelog entries.




## Test Steps

This PR should fail the check (see the last commit).
We'll need to remove the last commit before merging.



## Checklist

 - [n/a] I have updated or added any unit tests accordingly
 - [n/a] I have updated or added any integration tests accordingly
 - [n/a] I have updated or added any documentation accordingly

## Does this PR require extra reviews?
